apps/gr_satellites

```diff
@@ -292,6 +297,61 @@ class gr_satellites_top_block(gr.top_block):
     def setup_kiss_input(self):
         self.input = datasources.kiss_file_source(self.options.kiss_in)
 
+    def setup_sigmf_input(self):
+        if self.options.sigmf_in.endswith('sigmf'):
+            print(
+                "Failed to load SigMF archive (archives are not yet supported). "
```

Comment:
There is an inconsistent usage of double quotes (`"`) and simple quotes (`'`). In this PR. I believe that the gr-satellites code base only uses simple quotes to denote strings.